frontend/src/pages/NewProducts.js

```diff
@@ -12,7 +12,6 @@ function NewProductsPage() {
   return (
     <div>
       <button onClick={toggleCartHandler}>show cart</button>
-      <div><p>Do You Wanna know about new products?</p></div>
```

Comment:
please add another paragraph instead fo this paragraph

Reply:
as soon as possible